java/ql/src/Security/CWE/CWE-925/ImproperIntentVerification.qhelp

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE qhelp PUBLIC
+  "-//Semmle//qhelp//EN"
+  "qhelp.dtd">
+
+<qhelp>
+
+<overview>
+<p>
+When an android application uses a <code>BroadcastReciever</code> to receive Intents, 
+it is also able to receive explicit Intents that are sent drctly to it, egardless of its filter.
```

Comment:
The consensus is to use lower-case when not referring directly to the `Intent` class (in which case it should be enclosed in `<code>` tags.

Also, there's some typos.

```suggestion
When an android application uses a <code>BroadcastReciever</code> to receive intents, 
it is also able to receive explicit intents that are sent directly to it, regardless of its filter.
```